lnwallet/fee_estimator.go

```diff
@@ -301,6 +316,21 @@ func (b *BitcoindFeeEstimator) fetchEstimatePerVSize(
 		return 0, err
 	}
 
+	// Before proceeding, we'll make sure that this fee rate respects the
+	// minimum relay fee set on the backend node.
+	info, err := b.bitcoindConn.GetInfo()
+	if err != nil {
+		return 0, err
+	}
+
+	// If the estimated fee rate is lower than the minimum relay fee rate,
+	// then we'll use that.
+	if minFeeRate, err := btcutil.NewAmount(info.RelayFee); err == nil {
+		if satPerKB < minFeeRate {
```

Comment:
May be worth adding a debug logging statement here for future diagnostic purposes.

Reply:
Fixed.